source/blog/2019-02-15-the-ember-times-issue-85.md

```diff
@@ -1,19 +1,29 @@
 ---
 title: The Ember Times - Issue No. XX
-author: the crowd
+author: Chris Ng, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/xx/xx-the-ember-times-issue-XX.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Watch the Ember.js Documentary 🍿, <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [SECTION TITLE](#section-url)
+[## The Ember.js: The Documentary is now released! 🍿](https://twitter.com/honeypotio/status/1094974913725849606)
+
+<!--alex ignore spa-->
+JavaScript was not always the dominant force in the web. Today a lot of SPA features we see around the web was pioneered by Ember. Explore the story of why and how Ember.js came to be in [Ember.js: The Documentary](http://videos.honeypot.io/emberjs-documentary-2019/)!
+
+Here are some quick snippets from the film:
+
+> “JavaScript frameworks don’t have to be disposable software. Web applications don’t have to be disposable software. You don’t need to tell your manager we need to stop writing any new features for the next 6 months because it’s been 2 years and we need to rewrite – Ember is a symbol of the fact that that is a false dichotomy” - Tom Dale
+
+> “I really hope the best of Ember is ahead because I think front end engineering needs it … needs an example of something that is not trying to tell people that when something goes wrong it’s their own fault. So like something that tells you if you’re a beginner like this is for you” - Yehuda Katz
```

Comment:
```suggestion
> “I really hope the best of Ember is ahead because I think front end engineering needs it … needs an example of something that is not trying to tell people that when something goes wrong it’s their own fault. So like something that tells you if you’re a beginner like this is for you.” - Yehuda Katz
```